### DISCUSSION TITLE
Please help me with APK production error.

### DISCUSSION THREAD

**Discussion by etudelove** [airsdk/Adobe-Runtime-Support#3142] [Q&A]
![SDK-Component-Error](https://github.com/airsdk/Adobe-Runtime-Support/assets/52950369/f64ebb28-5a72-4cd2-bcc4-2074353547f1)

**Error Message**
SDK component at air/android/device/x86/Runtime.apk is out of date

**Even if I manipulate the adt.cfg file, try certain settings, try this, or that, I cannot resolve this error. I've been thinking about this problem for over a year.**

**Comment by ajwfrost** [comment]
Can you try re-extracting your AIR SDK, or downloading again into a new area for it? It sounds like the file mentioned is not the right one for the SDK - or, it's having problems trying to pull out the version information from the APK file mentioned there.

If you use AIR SDK Manager, perhaps set that up to download 50.2.4 into a new location, and then also go to the "Troubleshooting" page in that app whilst re-doing the build, to see if any further hints appear there?

thanks

**Reply by etudelove** [reply]
Thank you very much. There were special characters in the folder name in the AIR SDK folder path. This was resolved by deleting the special characters. Afterwards, a Java error occurred, so I changed the Java version to 11.0.22 and succeeded in creating the APK file without any errors. The error has been completely resolved. Thank you very much. thank you.
